ui/src/components/field/QasField.vue

```diff
@@ -178,12 +178,16 @@ export default {
       }
 
       return !this.value && (value === undefined || typeof value === 'string')
+    },
+
+    isNumberType () {
+      return this.field.type === 'number'
     }
   },
 
   methods: {
     emitValue (value) {
-      this.$emit('input', value)
+      this.$emit('input', this.isNumberType ? Number(value) : value)
```

Comment:
Por que não apenas?

```js
this.$emit('input', Number(value))
```

Reply:
Por que pode ter pessoas usando o retorno como String e ai nao quis atrapalhar isso, por isso se for number ele retorna um number, se nao continua como uma string.

Reply:
Ah, entendi. Foi preservado `string` então.